test/runtests.jl

```diff
@@ -17,6 +17,10 @@ end
     B = A + A
     @test B isa IntervalMatrix
 
+    # can subtract, TODO: add test
```

Comment:
Can we remove the TODO? Or what does it refer to?

Reply:
I copy-pasted it with the rest:

https://github.com/JuliaReach/IntervalMatrices.jl/blob/38f9826b26e45656358be7cc7f3330edd2ff568c/test/runtests.jl#L16-L22

Reply:
I added a proper test (also for addition).